WowPacketParserModule.V8_0_1_27101/Parsers/BattlegroundHandler.cs

```diff
@@ -95,5 +95,26 @@ public static void HandlePvPLogData(Packet packet)
             for (int i = 0; i < statisticsCount; i++)
                 ReadPvPMatchPlayerStatistics(packet, "Statistics", i);
         }
+
+        [Parser(Opcode.CMSG_BATTLEMASTER_JOIN_SKIRMISH)]
```

Comment:
this packet has changed between some patches, looking into it later today

Reply:
probably at the same time as plain CMSG_BATTLEMASTER_JOIN

Reply:
yes that packet change between patch, but I only check for 8.0.1 and 8.1.5